domains/vini.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "ThisPythonJS",
+        "email": "thispythonjs@gmail.com"
+    },
+    "records": {
+        "A": ["216.198.79.1"],
+        "TXT": "vc-domain-verify=vini.is-a.dev,25ac90d91fe7c07d69e9"
```

this is not where you add the TXT records, check the step 5 of this guide -> https://docs.is-a.dev/guides/vercel/

```suggestion
        "A": ["216.198.79.1"]
```